florist/api/launchers/local.py

```diff
@@ -60,7 +62,6 @@ def start_server(
             config=ServerConfig(num_rounds=n_server_rounds),
         )
         server.shutdown()
-        server.metrics_reporter.dump()
```

Comment:
Since we are dumping on every call to report I figured we can remove these?

Reply:
I guess, yes... I put it there just in case, it doesn't hurt, but I guess it's better to remove if it's not really necessary :)